Geometry/GEMGeometryBuilder/src/GEMGeometryBuilderFromDDD.cc

```diff
@@ -66,6 +67,7 @@ GEMGeometryBuilderFromDDD::build( GEMGeometry& theGeometry,
     if (detIdCh.layer() == 1){// only make superChambers when doing layer 1
       GEMSuperChamber *gemSuperChamber = buildSuperChamber(fv, detIdCh);
       theGeometry.add(gemSuperChamber);
+      superChambers.push_back(gemSuperChamber);
```

Comment:
It's a big dodgy to keep a list of writeable pointers after adding the super chambers to `theGeometry`. This technique works now, but future developers of the `GEMGeometry` class might assume that the `GemSuperChamber` objects stored in `GEMGeometry` cannot change since there is no interface that allows modification.
Perhaps the super chambers shouldn't be added to the `GEMGeometry` until after they are complete, but such an approach might have some side effects caused by changing the order of the objects stored in the `GEMGeometry` object.
Or a new method could be added to `GEMGeometry`:
`const std::vector<GEMSuperChamber*>& GEMGeometry::superChambersWriteable();`
to clearly allow writing to super chambers after they have been added.
This PR can go in as-is, but it would be good to consider whether this special pointer list is really the best approach.

Reply:
Okay, that's a good point. If we move this to after adding to the ring, we can avoid changing the superchambers after adding to theGeometry. I think the ordering should only make a difference in the GEMRing, but this isn't affected by this change.

Reply:
Adding the super chambers to `theGeometry` after they are modifed makes more sense. As long as there is no side effect from changing the order that the eta parts, GEM chambers, and super chambers are added, this approach is better.